package.json

```diff
@@ -71,9 +69,13 @@
     "time-grunt": "1.2.1"
   },
   "scripts": {
-    "test": "grunt test; grunt lint",
-    "start": "node --harmony-arrow-functions index.js",
-    "startdev": "iojs --harmony-arrow-functions index.js --development -c"
+    "build": "cp -r lib dist && babel --out-dir=dist index.js 'lib/**/*.js'",
```

Comment:
This means that `babel` would now be a required global package on all servers. We should warn everyone, and be sure to add it to the ansible playbook for node_server.